Sources/Imaging/Microsoft.Psi.Imaging.Windows/DepthImageCompressor.cs

```diff
@@ -27,12 +27,16 @@ public DepthImageCompressor(DepthCompressionMethod depthCompressionMethod)
             {
                 case DepthCompressionMethod.Png:
                     this.encoder = new DepthImageToPngStreamEncoder();
+                    this.decoder = new DepthImageFromStreamDecoder();
+                    break;
+                case DepthCompressionMethod.Tiff:
+                    this.encoder = new DepthImageToTiffStreamEncoder();
+                    this.decoder = new DepthImageFromTiffStreamDecoder();
```

Comment:
Should be able to just assign the `this.encoder` and use the generic `DepthImageFromStreamDecoder` per the original code. Unless this somehow didn't work?

Reply:
I reverted the change and only added the encoders.